### DISCUSSION TITLE
Selecting environment manually?

### DISCUSSION THREAD

**Discussion by Porter-dm** [community/community#37916] [Actions] [answered]
I have a deployment workflow that I want to deploy to multiple environments, each separately.  (i.e., I might deploy to dev today, uat tomorrow, and prod next week).

Is it possible to set up a workflow so that when it is triggered via `workflow_dispatch`, I can choose which environment it will run in?

**Comment by airtower-luna** [comment, marked as answer]
You can [provide *inputs* with the event](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#providing-inputs), including an environment. You can then use those inputs for whatever you need, note that there's a special input type for environments.
